test/functions_test.rb

```diff
@@ -141,8 +141,9 @@ def sass_return_path(path)
         Script::String.new("#{path.value}", :string)
       end
 
-      def optional_arguments(path, optional = "bar")
-        Script::String.new("#{path.value}/#{optional}", :string)
+      def optional_arguments(path, optional = nil)
+        optional ||= Script::String.new("bar")
```

Comment:
@edward this was the only functionality that really changed.  I believe for optional values we should be specifying them as script/value objects.  Might be breaking change for you?